src/main/scala/devices/gpio/GPIO.scala

```diff
@@ -141,14 +144,20 @@ abstract class GPIO(busWidthBytes: Int, c: GPIOParams)(implicit p: Parameters)
     GPIOCtrlRegs.iof_sel   -> iofSelFields,
     GPIOCtrlRegs.drive     -> Seq(RegField(c.width, dsReg,
                                   RegFieldDesc("ds","Pin drive strength selection", reset=Some(0)))),
+    GPIOCtrlRegs.drive1    -> Seq(RegField(c.width, ds1Reg,
+                                  RegFieldDesc("ds1","Pin drive strength selection", reset=Some(0)))),
```

Comment:
Instead of hardcoding `drive1` to exist, can we create a parameter which describes the width of the register?